index.js

```diff
@@ -1,67 +1,200 @@
-const express = require('express');
-const morgan = require('morgan');
-const app = express();
+const express = require('express'),
+bodyParser = require('body-parser'),
+morgan = require('morgan'),
+uuid = require('uuid')
+
+app = express();
+app.use(bodyParser.json());
 
 const options ={
     extensions: ['htm','html']
 }
-
-let tenMovies = [
+let users =[
+    {
+        username: 'Bodo73',
+        id: "1",
+        email:  'bodo@gmx.de',
+        favorites:[]
+    },
+    {
+        username: 'Bird',
+        id: "2",
+        email:  'peep@gmx.de',
+        favorites:[]
+    }
+];
+let movies = [
     {
         title: 'Black Panther',
-        director: ['Ryan Coogler']
+        director: 'Ryan Coogler',
```

Comment:
IMO a movie can have one or more directors and a director can direct in one or many movies. So the relationship can be one to many or many to many. To simplify this, let's model a relationship of `one to many` here?

Reply:
done